Description: /usr/include is a viable include dir
 Upstream removed /usr/include from the default include locations, which breaks
 the distutils test.  Revert of 6115fc52703baf07d4cfc629efb58866f0d16856.
Author: Dan Bungert <daniel.bungert@canonical.com>
Bug-Debian: https://bugs.debian.org/999812
Forwarded: no
Last-Update: 2021-11-11
--- numpy-1.21.2.orig/numpy/distutils/system_info.py
+++ numpy-1.21.2/numpy/distutils/system_info.py
@@ -327,7 +327,7 @@ else:
                                  '/opt/local/lib', '/sw/lib'], platform_bits)
     default_runtime_dirs = []
     default_include_dirs = ['/usr/local/include',
-                            '/opt/include',
+                            '/opt/include', '/usr/include',
                             # path of umfpack under macports
                             '/opt/local/include/ufsparse',
                             '/opt/local/include', '/sw/include',
@@ -336,7 +336,8 @@ else:
 
     default_x11_lib_dirs = libpaths(['/usr/X11R6/lib', '/usr/X11/lib',
                                      '/usr/lib'], platform_bits)
-    default_x11_include_dirs = ['/usr/X11R6/include', '/usr/X11/include']
+    default_x11_include_dirs = ['/usr/X11R6/include', '/usr/X11/include'
+                                '/usr/include']
 
     if os.path.exists('/usr/lib/X11'):
         globbed_x11_dir = glob('/usr/lib/*/libX11.so')
